acceptance/bundle/user_agent/verify.py

```diff
@@ -25,7 +25,7 @@ def check_user_agent(fname):
     if ".terraform." in fname:
         expected = "engine/terraform"
     elif ".direct" in fname:
-        expected = "engine/direct-exp"
+        expected = "engine/direct"
```

Comment:
Another part still refers to `direct-exp`.

Reply:
Which part? Note, that env var is still direct-exp.